webserver/views/api/v1/core.py

```diff
@@ -10,6 +10,19 @@
 bp_core = Blueprint('api_v1_core', __name__)
 
 
+@bp_core.route("/datasets", methods=["POST"])
```

Comment:
I actually already implemented this submission part in my PR with dataset API. https://github.com/metabrainz/acousticbrainz-server/pull/182/files#diff-f8f1fe605fbc564266894db22f5b32ffR23 Perhaps we can merge some of the parts that were implemented and you can just reuse them. @alastair, what do you think? I can modify the PR to include only endpoints that are done.